bindings/Cargo.toml

```diff
@@ -44,7 +44,7 @@ uuid = {version = "0.8", features = ["v4"]}
 futures = "0.3.5"
 derive-new = "0.5"
 #Python bindings
-pyo3 = { features = ["extension-module"], optional = true, version = "0.11.0" }
+pyo3 = { features = ["extension-module"], optional = true, version = "0.13.0" }
```

Comment:
Is pyo3 upgrade related to the connection leak issue ? If not could we have it as a separate PR?